Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/ui/homecards/cards/SevenDayRValueCard.kt

```diff
@@ -32,12 +38,38 @@ class SevenDayRValueCard(parent: ViewGroup) :
         }
 
         with(item.stats as SevenDayRValue) {
+
+            sevenDayRValueContainer.contentDescription =
+                buildAccessibilityStringForSevenDayRValueCard(item.stats, reproductionNumber)
+
             primaryLabel.text = getPrimaryLabel(context)
             primaryValue.text = getLocalizedSpannableString(
                 context,
                 formatStatisticalValue(context, reproductionNumber.value, reproductionNumber.decimals)
             )
+
+            primaryValue.contentDescription = StringBuilder()
+                .appendWithWhiteSpace(context.getString(R.string.statistics_title_reproduction))
+                .appendWithWhiteSpace(getPrimaryLabel(context))
+                .appendWithWhiteSpace(formatStatisticalValue(context, reproductionNumber.value, reproductionNumber.decimals))
+                .append(getContentDescriptionForTrends(context, reproductionNumber.trend))
+
             trendArrow.setTrend(reproductionNumber.trend, reproductionNumber.trendSemantic)
         }
     }
+
+    private fun buildAccessibilityStringForSevenDayRValueCard(
+        item: StatsItem,
+        reproductionNumber: KeyFigureCardOuterClass.KeyFigure
+    ): StringBuilder {
+
+        return StringBuilder()
+            .appendWithWhiteSpace(context.getString(R.string.accessibility_statistics_card_announcement))
+            .appendWithLineBreak(context.getString(R.string.statistics_title_reproduction))
+            .appendWithWhiteSpace(item.getPrimaryLabel(context))
+            .appendWithWhiteSpace(formatStatisticalValue(context, reproductionNumber.value, reproductionNumber.decimals))
+            .appendWithWhiteSpace(context.getString(R.string.statistics_card_incidence_value_description))
```

Comment:
This should be `statistics_reproduction_average`

Reply:
Good catch! Is changed ;)